### PR TITLE
Organ wise counts in all organs report

### PR DESCRIPTION
**Pull request by vutukuriajay2241**
Successfully completed creating a organ wise for all organs report when "showall AS" toggle is on.

### CODE REVIEW

File: projects/v2/src/app/components/report/report.component.html [hunk 1]
```diff
@@ -66,29 +148,94 @@
           <div class="h5 mt-3 mb-3">Details</div>
 
           <div class="details pr-4">
-            <p class="text-muted mb-3"><small class="font-weight-normal">Alphabetically sorted listings of all AS, CT &
-                B</small></p>
+            <div *ngIf="SheetConfig.show_all_AS && currentSheet.name == 'all'">
+              <p class="text-muted mb-3">
+                <small class="font-weight-normal">Organ Wise</small>
```
[the remark this answers is not in the document]
Comment (vutukuriajay2241) [reply projects/v2/src/app/components/report/report.component.html]: Done

File: projects/v2/src/app/components/report/report.component.html [hunk 2]
```diff
@@ -66,29 +148,94 @@
...
+              </p>
+
+              <table
+                mat-table
+                [dataSource]="countsByOrgan"
+                class="mat-elevation-z8 count_by_organ_table"
+              >
+              <caption class="sr-only">Organ Wise Count for all organs</caption>
```
[the remark this answers is not in the document]
Comment (vutukuriajay2241) [reply projects/v2/src/app/components/report/report.component.html]: Done

File: projects/v2/src/app/components/report/report.component.html [hunk 3]
```diff
@@ -66,29 +148,94 @@
...
+                <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
+                <tr
+                  mat-row
+                  *matRowDef="let row; columns: displayedColumns"
+                ></tr>
+              </table>
+            </div>
+            <p class="text-muted my-3">
+              <small class="font-weight-normal"
+                >Alphabetically sorted listings of all AS, CT & B</small
```
[the remark this answers is not in the document]
Comment (vutukuriajay2241) [reply projects/v2/src/app/components/report/report.component.html]: Done

File: projects/v2/src/app/components/report/report.component.html [hunk 4]
```diff
@@ -11,52 +16,129 @@
...
+            <p class="text-muted mb-3">
+              <small class="font-weight-bolder">UNIQUE ENTITIES</small>
+            </p>
+            <div
+              class="
+                d-flex
+                justify-content-between
+                align-content-center
+                pr-5
+                h6
```
[the remark this answers is not in the document]
Comment (vutukuriajay2241) [reply projects/v2/src/app/components/report/report.component.html:31]: Thats the part of prettier formatter.

**Comment by bherr2** [comment]
I don't normally worry about the code complexity smells that sonarcloud flags, but ... complexity 64 is .. a lot! Try to reduce the complexity of [this one](https://sonarcloud.io/code?id=hubmapconsortium_ccf-asct-reporter&pullRequest=106&selected=hubmapconsortium_ccf-asct-reporter%3Aprojects%2Fv2%2Fsrc%2Fapp%2Fcomponents%2Freport%2Freport.service.ts&line=114)

**Review by bherr2 — APPROVED** [review]

**Pull request merged**